plugins/parquet/parquetembed.cpp

```diff
@@ -956,12 +956,42 @@ __int64 ParquetRowBuilder::getCurrIntValue(const RtlFieldInfo *field)
         {
             __int64 myint64 = 0;
             auto scalar = getCurrView(field);
-            handleDeserializeOutcome(tokenDeserializer.deserialize(scalar.data(), myint64), "signed", scalar.data());
+            handleDeserializeOutcome(tokenDeserializer.deserialize(StringBuffer(scalar.size(), scalar.data()).str(), myint64), "signed", scalar.data());
```

Comment:
style: we don't normally use inline temporary constructors like this, it tends to make the code less clear. It would be clearer to declare on a proceeding line.
Also, given the scalar is a std::string_view using a std::string vs a StringBuffer would seem more in keeping.

Reply:
Removed inline constructor and changed container to std::string.